src/HyperFormula.ts

```diff
@@ -423,7 +423,7 @@ export class HyperFormula implements TypedEmitter {
   }
 
   /**
-   * Registers a function with a given id if such exists in a plugin
+   * Registers a function with a given id if such exists in a plugin (as a function or an alias).
```

Comment:
I'm not sure how to do it. An example would be nice. We can throw a new error here if I understood the code correctly. And the description is now harder to understand. "Registers a function .. as a function". Should we rephrase the whole description? CC @scarletfog

Reply:
I think that `registerFunctionPlugin` method has changed internally as well. Not sure how do we handle `unregister*` methods when it comes to aliases @izulin?

Reply:
Aliases are registered just as functions, so we are able to unregister them exactly in the same way.

Reply:
Ok, I agree that the explanation is a little confusing. I reverted it back.